pxe-formula/pxe-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jan 14 13:25:19 UTC 2025 - Ondrej Holecek <oholecek@suse.com>
```

Comment:
no hard preferences here... not sure if you have specific reasons for an additional changelog entry,  but  from my POV you could edit the other one and just use something like:

~~~
Tue Jan 14 10:38:56 UTC 2025 - Ondrej Holecek <oholecek@suse.com>

- Update to version 0.3.0
  * Fix license snippets in source to be correctly GPL-2.0+
  * spec file updates

~~~

Reply:
but given that you have everything ready, feel free to also ignore this comment